docs/user_guide/HOWTO/Manage_TPM.rst

```diff
@@ -362,4 +445,25 @@ Trusted Boot Debugging Tips and Warnings
    Be careful updating the ``kernel`` packages and rebuilding the ``initramfs``
    (or running ``dracut``).
 
+HIRS
+----
+
+The SIMP hirs_provisioner module installs and configures the :term:`HIRS` TPM
+Provisoner on specified systems. An Attestation Certificate Authority (ACA)
```

Comment:
Would be helpful if there was an ACA term in the glossary.

Reply:
The term is pretty specific to HIRS and is defined and detailed on their website which is linked in the glossary.